django_project/lesson/static/css/worksheet_detail.css

```diff
@@ -0,0 +1,100 @@
+/* /!\ Copy/paste from the organisation_detail.css file /!\ */
```

Comment:
Is this file even needed - if you have it all in another css it will just fetch it from there assuming you have it in the pipeline.....

Reply:
I don't know if we should keep app self contained or if can share CSS from another app. Ok, I will just include it.

Reply:
You shouldnt need to include it other do

```
{% compressed_css 'project' %}
```

- pipeline will do the rest I think